openapi/paths/tags@{tag}@customers.yaml

```
@@ -69,7 +69,7 @@ delete:
                 type: string
   responses:
     '204':
-      description: Customers untagging process has been scheduled.
+      description: Customers untagging
```

suggestion: for readability
```suggestion
      description: Untagging customers 
```